src/server/api/general.js

```diff
@@ -573,6 +573,71 @@ export default function useGeneralApi(app) {
             this.status = 500;
         }
     });
+
+    router.post('/isTosAccepted', koaBody, function*() {
+        const params = this.request.body;
+        const { csrf } =
+            typeof params === 'string' ? JSON.parse(params) : params;
+        if (!checkCSRF(this, csrf)) return;
+
+        if (!this.session.a) {
+            this.body = 'missing username';
+            this.status = 500;
+            return;
+        }
+
+        try {
+            const res = yield api.signedCallAsync(
+                'conveyor.get_tags_for_user',
+                [this.session.a],
+                config.get('conveyor_username'),
+                config.get('conveyor_posting_wif')
+            );
+
+            this.body = JSON.stringify(!!res.includes('accepted_tos'));
```

Comment:
No need to coax this into a bool

Reply:
good point; relic of another iteration of this code.